modules/cloudbeta/modules/resource-manager/pages/workspaces/writeread_readonly.adoc

```diff
@@ -22,6 +22,22 @@ Read-only workspaces provide improved performance and scalability for read opera
 [NOTE]
 ====
 Current RO workspace requires manual sync up with the RW workspace to catch up with the latest change.
+
+In RO workspace, the following operations are forbidden:
+
+1. run write queries
```

Comment:
Can we make the forbidden operations in categories:
Database Operations,
System Administration Commands,
Schema Management, 
Graph Management, 
Data Modification, 
Loading Jobs
Backup and Restore,